babel.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         ],
         "@babel/react",
       ],
+      plugins: ["dynamic-import-node"],
```

Comment:
Is this only needed for tests? If so, can you move it into that env?

Reply:
It is already :-)